### PR TITLE
Add triagebot mentions entry for simd intrinsics

### LINKED ISSUE #119119: Triagebot ping for changes to library/core/src/intrinsics/simd.rs
https://github.com/rust-lang/rust/pull/118853 will introduce this submodule. Any changes here need all codegen backends to implement them. And portable-simd likely wants to know about it. As such I propose adding a ping for any change to this file mentioning me, @antoyo, @GuillaumeGomez and @rust-lang/project-portable-simd.

### PR DESCRIPTION
**Pull request by bjorn3**
Fixes https://github.com/rust-lang/rust/issues/119119

cc @antoyo, @GuillaumeGomez, @rust-lang/project-portable-simd

### CODE REVIEW

**Comment by bjorn3** [comment]
For portable-simd I only added the people already in the mentions entry for portable-simd under the assumption that the rest doesn't want to get notified. Please tell me if you want to be mentioned anyway.

**Review by GuillaumeGomez — APPROVED** [review]

File: triagebot.toml [hunk 1]
```diff
@@ -492,6 +492,13 @@ cc = ["@Nadrieril"]
 message = "Some changes might have occurred in exhaustiveness checking"
 cc = ["@Nadrieril"]
 
+[mentions."library/core/src/intrinsics/simd.rs"]
+message = """
+Some changes occurred to the platform-builtins intrinsics. Make sure all
+codegen backends as well as portable-simd get adapted for the changes.
```
Comment (programmerjake) [inline triagebot.toml:498]: maybe phrase this so people creating PRs aren't under the impression that they have to update every backend by themselves? iirc gcc and cranelift aren't supposed to block PRs...

Comment (Mark-Simulacrum) [reply triagebot.toml:498]: Yeah, I think we should update the text here, or make a T-compiler decision if we do want to block PRs. My guess is that we're not ready to block on making the backends work yet.

Comment (bjorn3) [reply triagebot.toml]: Done

**Comment by Nilstrieb** [comment]
sounds good
@bors r+ rollup

**Pull request merged**